lesson4/lesson4whilestudying/lesson4whilestudying/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace lesson4whilestudying
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string moving = Convert.ToString(Console.ReadLine());
+            try
+            {
+                char move = Convert.ToChar(moving);
+                FigureMove(move);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("There is more than one movement");
```

Comment:
It's really cool that you decided to add special case!!! I like it.
But it's not good way to handle such cases with exceptions as exceptions usually describe the beheviour that you don't expect and don't know how to handle it.
So in this situation the better way would be to check how much symbols you got and then use either the first or the second algorithm.